common/patches/rof2.cpp

```diff
@@ -6358,9 +6358,19 @@ namespace RoF2
 		//sprintf(hdr.unknown000, "06e0002Y1W00");
 		strn0cpy(hdr.unknown000, fmt::format("{:016}\0", inst->GetSerialNumber()).c_str(),sizeof(hdr.unknown000));
 
-		hdr.stacksize =
-			item->ID == PARCEL_MONEY_ITEM_ID ? inst->GetPrice() : (inst->IsStackable() ? ((inst->GetCharges() > 1000)
-				? 0xFFFFFFFF : inst->GetCharges()) : 1);
+		hdr.stacksize = (
```

Comment:
This is not the most readable doing nested ternaries like this and is very hard to follow.

Suggest using something like this 

```cpp
hdr.stacksize = 1;
if (item->ID == PARCEL_MONEY_ITEM_ID) {
    hdr.stacksize = inst->GetPrice();
} else if (inst->IsStackable()) {
    if (inst->GetCharges() > std::numeric_limits<int16>::max()) {
        hdr.stacksize = std::numeric_limits<uint32>::max();
    } else {
        hdr.stacksize = inst->GetCharges();
    }
}
```